hb_config.sample.py

```diff
@@ -71,3 +71,6 @@
 # of looking for plain strings, these stings are used to flexibly
 # search for templates.
 skip_templates = ["blocked"]
+
+# Wait at least this long between sending invitations.
```

Comment:
I would add that it is in second and that it is between inviting each individual (as compared to between each run or between a second invitation to the same person)